### PR TITLE
Update Frame shortcut

### PR DESCRIPTION
**Pull request by DLu**

### CODE REVIEW

**Review by clalancette — COMMENTED** [review]
Just so I understand; this changes no logic, but just consolidates some duplicated lines of code into a method.  Is that correct?

**Comment by DLu** [comment]
Correct

File: rviz_common/include/rviz_common/display.hpp [hunk 1]
```diff
@@ -285,6 +285,12 @@ class RVIZ_COMMON_PUBLIC Display : public rviz_common::properties::BoolProperty
   properties::Property *
   findProperty(const QString & name);
 
+  /// Get the latest transform to the frame and update the scene node. Return true on success.
+  bool updateFrame(const std::string& frame);
```
Comment (clalancette) [inline rviz_common/include/rviz_common/display.hpp]:
```suggestion
  bool updateFrame(const std::string & frame);
```

**Review by clalancette — CHANGES_REQUESTED** [review]
I've left one style nit to fix, otherwise this looks good to me.

**Review by clalancette — APPROVED** [review]

**Comment by clalancette** [comment]
CI:
* Linux [Build Status](http://ci.ros2.org/job/ci_linux/18263/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/12799/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/18940/)

**Comment by DLu** [comment]
Backport to foxy/humble?

**Comment by clalancette** [comment]
> Backport to foxy/humble?

I think we could; it doesn't really change API/ABI.  On the other hand, it seems like a simple cleanup, so I'm not sure it is worth it.  What's your interest in the backport?

**Comment by DLu** [comment]
Cleaning up [my new tutorial](https://github.com/ros2/ros2_documentation/pull/3384/files#diff-9fd71dd84295290656bf218e71ff024d71efdf0cfceadffc44370eb153cb20a2R233)

**Comment by clalancette** [comment]
I'll open up the backport to Humble, though I want to have some soak time in Rolling/Iron before we merge this in.

I'm less inclined to do it for Foxy since we are so close to its EOL.

**Comment by clalancette** [comment]
@Mergifyio backport humble

**Comment by DLu** [comment]
Thanks @clalancette ! Thanks @mergifyio!

**Pull request merged**